tools/api-markdown-documenter/src/api-item-transforms/configuration/Hierarchy.ts

```diff
@@ -0,0 +1,407 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { type ApiItem, ApiItemKind, type ApiPackage } from "@microsoft/api-extractor-model";
+
+import {
+	getApiItemKind,
+	getUnscopedPackageName,
+	type ValidApiItemKind,
+} from "../../utilities/index.js";
+import { createQualifiedDocumentNameForApiItem } from "../ApiItemTransformUtilities.js";
+
+/**
+ * Kind of documentation suite hierarchy.
+ *
+ * @public
+ */
+export enum HierarchyKind {
```

Comment:
`export const enum`? Doesn't seem like we need access to the enum keys at runtime. Not really a big deal for this tool as compared to Fluid packages, but using the preferred pattern for the codebase seems like a good idea. Quick experiment in TSPlayground suggests the compiler doesn't mind them in generics like we use below, but I'm curious if it really works at runtime 🤔 .

Reply:
I'll give it a shot.

Reply:
Actually, other enums in this library are just standard enums. I think I'll err on the side of internal consistency for now.